<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-05-05T09:43:49Z" name="inetBanking_v1" finished-at="2021-05-05T09:44:11Z" duration-ms="21739">
    <groups>
    </groups>
    <test started-at="2021-05-05T09:43:49Z" name="Bankingweb Test" finished-at="2021-05-05T09:44:11Z" duration-ms="21739">
      <class name="com.inetBanking_testCases.TC_AddCustomerTest_003">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.inetBanking_testCases.TC_AddCustomerTest_003@609bcfb6]" started-at="2021-05-05T15:13:49Z" name="setup" finished-at="2021-05-05T15:13:57Z" duration-ms="8080" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="addNewCustomer()[pri:0, instance:com.inetBanking_testCases.TC_AddCustomerTest_003@609bcfb6]" started-at="2021-05-05T15:13:57Z" name="addNewCustomer" finished-at="2021-05-05T15:14:10Z" duration-ms="12698" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.inetBanking_testCases.TC_AddCustomerTest_003.addNewCustomer(TC_AddCustomerTest_003.java:56)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.inetBanking_testCases.TC_AddCustomerTest_003@609bcfb6]" started-at="2021-05-05T15:14:10Z" name="teardown" finished-at="2021-05-05T15:14:11Z" duration-ms="753" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.inetBanking_testCases.TC_AddCustomerTest_003 -->
    </test> <!-- Bankingweb Test -->
  </suite> <!-- inetBanking_v1 -->
</testng-results>
